caesars-palace/scripts/scripts.js

```diff
@@ -209,18 +209,26 @@ export function buildEllipsis(text, width, maxVisibleLines, suffix, options = {}
   };
 }
 
+/**
+ * Checks if a string contains only numbers. Needed for product details
+ * @returns {Boolean} true/false
+ */
+export function containsOnlyNumbers(str) {
+  return !(/[a-zA-Z]/.test(str));
```

Comment:
This only ensures the string does not contain alpha characters? Would checking for what you actually want instead of the negative case be better?

eg.
```
/^[0-9]+$/
```